java/runtime/src/main/java/org/ray/runtime/util/BinaryFileUtil.java

```diff
@@ -45,4 +47,23 @@ public static File getFile(String destDir, String fileName) {
     }
     return file;
   }
+
+  public static File getFileSafe(String destDir, String fileName) {
```

Comment:
`getFile` seems unused now, could you remove it?

Reply:
Also comment on this method.

Reply:
done